WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -397,6 +403,18 @@ extension UIImage {
         return ellipsisImage.imageWithTintColor(tintColor)!
     }
 
+    /// Payment celebration
+    ///
+    static var paymentCelebrationImage: UIImage {
```

Comment:
Optional. The image shows nothing payments related. Consider renaming static var and file to simply celebration